### ISSUE TITLE
Getter and property can have the same name

### ISSUE THREAD

**Issue by ariya** [eslint/espree#179]
Test case:

```js
espree.parse('var foo = { bar: function() {}, get bar() {} }', { ecmaFeatures: { objectLiteralDuplicateProperties: true } })
```

Actual: Error: Line 1: Object literal may not have data and accessor property with the same name

Expected: a valid syntax tree.

**Comment by nzakas** [comment]
Fixed in 3.0.0.
